docs/content.zh/docs/ops/metrics.md

```diff
@@ -916,12 +916,17 @@ Metrics related to data exchange between task executors using netty network comm
       <td>Gauge</td>
     </tr>
     <tr>
-      <th rowspan="18">Task</th>
-      <td rowspan="4">Shuffle.Netty.Input.Buffers</td>
+      <th rowspan="20">Task</th>
+      <td rowspan="5">Shuffle.Netty.Input.Buffers</td>
       <td>inputQueueLength</td>
       <td>The number of queued input buffers.</td>
       <td>Gauge</td>
     </tr>
+    <tr>
+      <td>inputQueueSize</td>
+      <td>The real size of queued input buffers in bytes. The size for local input channels is only estimated.</td>
```

Comment:
On a second thought. Shouldn't we actually ignore local input channels? 

1. It would be kind of consistent with the other metrics
2. Otherwise, don't we double account those bytes on the output and on the input?

Reply:
In fact, you are right we count bytes twice for the local input channel but we do the same thing for `outputQueueLength`/`inputQueueLength`. But maybe it is not a bad idea to always return zero for local input channels since they take the data directly from output.